spec/integ/crypto/verification.spec.ts

```diff
@@ -980,12 +980,43 @@ describe.each(Object.entries(CRYPTO_BACKENDS))("verification (%s)", (backend: st
             });
         }
 
+        it("Verification request not found", async () => {
+            // Expect to not find any verification request
+            const request = aliceClient.getCrypto()!.findVerificationRequestDMInProgress(TEST_ROOM_ID, "@bob:xyz");
+            expect(request).not.toBeDefined();
```

Comment:
Jest also provides the opposite matcher out of the box, shouldn't we use `toBeUndefined` instead? 
Feels like it reads better, but also feel free to leave this minor nitpick aside

Reply:
yup, good idea

```suggestion
            expect(request).toBeUndefined();
```